src/App.js

```diff
@@ -1,6 +1,7 @@
 import logo from './logo.svg';
 import './App.css';
 import { MichaelNguyen } from './components/MichaelNguyen/MichaelNguyen';
+import { JuliaSharnina } from './components/MichaelNguyen/JuliaSharnina';
```

Comment:
You created the new file in the `components/MichaelNguyen` folder, it might be clearer to put it in a new folder `components/JuliaSharnia` instead

Reply:
Yes, sure